@joystream/types can't install

```
$ yarn add @joystream/types
yarn add v1.15.2
[1/5] Validating package.json...
[2/5] Resolving packages...
[3/5] Fetching packages...
info fsevents@1.2.9: The platform "linux" is incompatible with this module.
info "fsevents@1.2.9" is an optional dependency and failed compatibility check. Excluding it from installation.
[4/5] Linking dependencies...
[5/5] Building fresh packages...
error /home/jens/src/joy/storage-node-joystream/node_modules/@joystream/types: Command failed.
> @joystream/types@0.1.3 build /home/jens/src/joy/storage-node-joystream/node_modules/@joystream/types
> tsc --build tsconfig.json

src/index.ts(11,10): error TS2742: The inferred type of 'none' cannot be named without a reference to '@joystream/types/node_modules/@polkadot/types/types'. This is likely not portable. A type annotation is necessary.
src/index.ts(15,10): error TS2742: The inferred type of 'some' cannot be named without a reference to '@joystream/types/node_modules/@polkadot/types/types'. This is likely not portable. A type annotation is necessary.
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! @joystream/types@0.1.3 build: `tsc --build tsconfig.json`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the @joystream/types@0.1.3 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

I've been trying to debug it a little, and found that if I symlink the `joy-types` to the storage node's `packages/types`, the command works.

So I suspect something in the `tsconfig.json`, maybe the type roots are wrong?

I also tried with npm v6.4.1, node v10.15.3

```
MokoBook:tmp mokhtar$ npm install @joystream/types

> secp256k1@3.7.0 install /Users/mokhtar/tmp/node_modules/secp256k1
> npm run rebuild || echo "Secp256k1 bindings compilation fail. Pure JS implementation will be used."


> secp256k1@3.7.0 rebuild /Users/mokhtar/tmp/node_modules/secp256k1
> node-gyp rebuild

  CXX(target) Release/obj.target/secp256k1/src/addon.o
  CXX(target) Release/obj.target/secp256k1/src/privatekey.o
../src/privatekey.cc:73:3: warning: ignoring return value of function declared with 'warn_unused_result' attribute
      [-Wunused-result]
  secp256k1_ec_privkey_negate(secp256k1ctx, &private_key[0]);
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
In file included from ../src/privatekey.cc:4:
In file included from ../src/secp256k1-src/src/scalar_impl.h:10:
In file included from ../src/secp256k1-src/src/group.h:11:
../src/secp256k1-src/src/field.h:36:13: warning: unused function 'secp256k1_fe_normalize' [-Wunused-function]
static void secp256k1_fe_normalize(secp256k1_fe *r);
            ^
../src/secp256k1-src/src/field.h:39:13: warning: unused function 'secp256k1_fe_normalize_weak' [-Wunused-function]
static void secp256k1_fe_normalize_weak(secp256k1_fe *r);
            ^
.
.
SNIP  ...... secp256k1-src building.....
.
.

      [-Wunused-function]
static int secp256k1_ecmult_strauss_batch_single(const secp256k1_ecmult_context *actx, secp256k1_scratch *scratch, secp256...
           ^
../src/secp256k1-src/src/ecmult_impl.h:1042:12: warning: unused function 'secp256k1_ecmult_pippenger_batch_single'
      [-Wunused-function]
static int secp256k1_ecmult_pippenger_batch_single(const secp256k1_ecmult_context *actx, secp256k1_scratch *scratch, secp2...
           ^
7 warnings generated.
  CC(target) Release/obj.target/secp256k1/src/secp256k1-src/contrib/lax_der_parsing.o
  CC(target) Release/obj.target/secp256k1/src/secp256k1-src/contrib/lax_der_privatekey_parsing.o
  SOLINK_MODULE(target) Release/secp256k1.node

> @joystream/types@0.1.3 postinstall /Users/mokhtar/tmp/node_modules/@joystream/types
> npm run build


> @joystream/types@0.1.3 build /Users/mokhtar/tmp/node_modules/@joystream/types
> tsc --build tsconfig.json

npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN tmp@1.0.0 No description
npm WARN tmp@1.0.0 No repository field.

+ @joystream/types@0.1.3
added 66 packages from 96 contributors and audited 848 packages in 26.37s
found 0 vulnerabilities
```

--- a/packages/joy-types/tsconfig.json
+++ b/packages/joy-types/tsconfig.json
@@ -11,12 +11,7 @@
     "esModuleInterop": true,                  /* Enables emit interoperability between CommonJS and ES Modules via creation of namespace objects for all imports. Implies 'allowSyntheticDefaultImports'. */
     "experimentalDecorators": true,           /* Enables experimental support for ES7 decorators. */
     "declaration": true,
-    "outDir": "lib",
-    "typeRoots": [
-      "./node_modules/@polkadot/ts",
-      "./node_modules/@types",
-      "./lib"
-    ]
+    "outDir": "lib"
   },
   "include": [
     "src/*.ts"

